include/kframework/evm.md

```diff
@@ -925,9 +920,9 @@ NOTE: We have to call the opcode `OR` by `EVMOR` instead, because K has trouble
 
     syntax BinStackOp ::= "SHL" | "SHR" | "SAR"
  // -------------------------------------------
-    rule <k> SHL W0 W1 => W1 <<Word  W0 ~> #push ... </k>
-    rule <k> SHR W0 W1 => W1 >>Word  W0 ~> #push ... </k>
-    rule <k> SAR W0 W1 => W1 >>sWord W0 ~> #push ... </k>
+    rule <k> SHL W0 W1 => W1 <<Word  W0 ~> #push ... </k> requires W0 >=Int 0 [preserves-definedness]
+    rule <k> SHR W0 W1 => W1 >>Word  W0 ~> #push ... </k> requires W0 >=Int 0 [preserves-definedness]
+    rule <k> SAR W0 W1 => W1 >>sWord W0 ~> #push ... </k> requires W0 >=Int 0 [preserves-definedness]
```

Comment:
Can we instead make `<<Word`, `>>Word`, and `>>sWord` total? Was there anything blocking that?

We could basically just have an `owise` case for each rule that sends the result to `0` if it's not defined currently.